internal/connect/hwinfo.go

```diff
@@ -100,8 +100,9 @@ func hwinfoS390(hw *hwinfo) error {
 	if err != nil {
 		return err
 	}
-	hw.UUID = string(rvuOut)
-	if hw.UUID == "" {
+	if !strings.HasPrefix(string(rvuOut), "help:") {
```

Comment:
Please instead fix this in the uuid() function.

Reply:
The way uuid is found for x86 and arm is completely different than s390. So I decided to put them in different functions. I have added some comments to make this more clear. Also now it checks for a valid uuid and returns "" for anything else.

Reply:
Looks good now.